cadence/contracts/LendingComptroller.cdc

```diff
@@ -217,7 +215,7 @@ pub contract LendingComptroller {
         ///
         /// Since borrower would decrease his overall collateral ratio across all markets, safety check is important.
         ///
-        pub fun redeemAllowed(
+        access(all) fun redeemAllowed(
```

Comment:
`view`?

Reply:
Cannot be made view as non-view operation performed in `callerAllowed()`